examples/platforms/cc2650/random.c

```diff
@@ -145,6 +142,11 @@ ThreadError otPlatRandomSecureGet(uint16_t aInputLength, uint8_t *aOutput, uint1
  */
 int mbedtls_hardware_poll(void *data, unsigned char *aOutput, size_t aLen, size_t *oLen)
 {
+    ThreadError error;
+
     (void)data;
-    return TRNGPoll(aOutput, aLen, oLen);
+    error = TRNGPoll(aOutput, aLen);
```

Comment:
`TRNGPoll` could return an error from `TRNGStatusGet` on line 102. Instead we just spin in a busy while loop until the entropy pool is full enough, if it is ever full enough. 

This is probably lazy coding on my part. I figured that since OpenThread had no mechanism to reset the rng; any error polling would be fatal. A better design would be to implement a timeout to the loop on 102 and check the bit-field returned.